packages/plugins/rubygem-foreman_bootdisk/rubygem-foreman_bootdisk.spec

```
@@ -8,8 +8,8 @@
 %global foreman_min_version 3.1
```

Could you update this to match https://github.com/theforeman/foreman_bootdisk/blob/v21.0.0/lib/foreman_bootdisk/engine.rb#L33
```suggestion
%global foreman_min_version 3.4
```